### ISSUE TITLE
how to make the training set,validation set ,test set of R-CNN?

### ISSUE THREAD

**Issue by wuxinhong** [BVLC/caffe#2250]
how to make the training set,validation set ,test set of R-CNN?  for this need which job to do?
R-CNN's training set,validation set ,test set are completely different to general classification  model 's training set,validation set ,test set, could anyone know ,thank you

**Comment by C0NPAQ** [comment]
Great ... the answer to this can be found nowhere, but probably only by implication reading documentation for 3 days. It is all I need. Would be nice if someone just gave a simple answer.
